accounts/views.py

```diff
@@ -729,6 +729,39 @@ def attribution(request):
     return render(request, 'accounts/attribution.html', tvars)
 
 
+@login_required
+def download_attribution(request):
+    content = {"csv": {"style": "csv", "pattern": "%s,%s,%s,%s\n"},
```

Comment:
I just had a check, we have 14000 sounds in freesound where the name has a comma in it. This means that it's absolutely necessary to use the csv library to write the csv version.
You could use a StringIO object to write the result to, and then get the value of that to send to the HttpResponse

Reply:
Done and done, and ready for push; and now onto the test...